Documentation/kubernetes/install/eks.rst

```diff
@@ -54,21 +54,12 @@ Create EKS Cluster
 
        kubectl -n kube-system set env ds aws-node AWS_VPC_K8S_CNI_EXTERNALSNAT=true
 
-#. Assign a fixed security identity to ``kube-dns`` by  adding the label ``io.cilium.fixed-identity: kube-dns``
+#. Restart kube-dns/coredns to ensure that it is being managed by Cilium.
```

Comment:
You should fix the README as well to be in sync with these new instructions - see commit https://github.com/cilium/cilium/commit/b1985eaf1a615a652a1d1c1978bc489873c6d9ab